python/heterocl/devices.py

```diff
@@ -275,7 +275,7 @@ def config(self, compile=None, mode=None, backend=None):
           self.tool = tool(compile, *option_table[compile]) 
         
         if mode: # check tool mode 
-          modes = ["sw_sim", "hw_sim", "hw_exe", "debug"]
```

Comment:
We can consider removing the `sw_exe` mode and add more options to configure the `hw_sim` mode.